### DISCUSSION TITLE
Position images freely

### DISCUSSION THREAD

**Discussion by CaneRandagio1983** [freeplane/freeplane#1311] [Ideas]
Currently with version 1.10.3 **images can only be placed under the relative node**.

**I need to insert images to the right or left of the node**, or **alternatively insert free images**, 
without connections and relative node.

This is currently not possible, it would be really useful to have this feature.

Thank you

**Comment by macmarrum** [comment]
A mind map, by design, consists of nodes. Any piece of information is stored in a node. Also images.

Having said that, it's possible to have a node which consists only of an image. The image will respect the rules of node placement, i.e. will be moved around depending on the layout / other nodes. The node can be made into a free node, giving the image all the benefits and limitations of a free node.

Here's an example
[FP-1311-position-images-freely.zip](https://github.com/freeplane/freeplane/files/12099930/FP-1311-position-images-freely.zip)

If you are looking for static placement of images in the mind map, you can use GIMP or similar to combine your pictures into a single image and then set it via `Format->Map background->Background Image…`. You can use `FP-1311-position-images-freely-files/background.png` from the zip file as an example background.

**Reply by CaneRandagio1983** [reply]
Thanks for the reply @macmarrum 
So just create a node with transparent edges and links, this is an acceptable solution.

Thanks again
